deploy.sh

```diff
@@ -51,6 +68,23 @@ if ! $found; then
 fi
 echo "Using AWS account $AWS_ACCOUNT" >&2
 
+export GCP_PROJECT_ID
+case "$DEPLOYMENT" in
+    #taskcluster-net) GCP_PROJECT_ID=linux64-builds;;
```

Comment:
I assume this is commented intentionally.

Reply:
It was, but probably unnecessarily.